tests/ert/unit_tests/services/test_storage_service.py

```diff
@@ -46,3 +49,34 @@ def test_that_service_can_be_started_with_existing_conn_info_json(tmp_path):
     with open(tmp_path / "storage_server.json", mode="w", encoding="utf-8") as f:
         json.dump(connection_info, f)
     StorageService.connect(project=tmp_path)
+
+
+@pytest.mark.integration_test
+def test_storage_logging(change_to_tmpdir):
+    """
+    This is a regression test for a bug where the storage service
+    would log everything twice
+    """
+
+    with StorageService.start_server(
+        verbose=True,
+        project=".",
+        parent_pid=os.getpid(),
+    ) as server:
+        server.wait_until_ready()
+
+        logfiles = glob.glob("api-log-storage*.txt")
+        assert len(logfiles) == 1, "Expected exactly one log file"
```

Comment:
Will this test fail prior to the change in storage_log.conf? The problem was duplicated log entries, not duplicated files..?

Reply:
(but logging configuration like this is typically not tested thorougly, so you might also skip the test if it becomes ugly)

Reply:
The regression is the three assert below this one

Reply:
Ah yes. 

But I think this test is too specific, it will start failing when we tune which log messages are written, and then represent a maintenance burden.

Reply:
I can make it only check for duplicate entries instead